src/store/modules/blocks/getters.js

```diff
@@ -1,4 +1,31 @@
 /**
  * Exporting Getters
  */
-export default {}
+export default {
+  /**
+   * TODO: Refactor this function
+   * getAverageBlockTime by calculating
+   * the list of the blocks on the state
+   * @param state
+   * @return {number}
+   */
+  getAverageBlockTime: function (state) {
+    const blockTimes = state.blocks.map(
+      (block, idx, blocks) => idx + 1 < blocks.length && blocks[idx].time - blocks[idx + 1].time
+    )
+    // no idea what this does
```

Comment:
it creates a new array every element of it is a difference between creation times of two corresponding blocks